packages/firebase_core/firebase_core/windows/firebase_core_plugin.cpp

```diff
@@ -38,6 +42,26 @@ void FirebaseCorePlugin::RegisterWithRegistrar(
   registrar->AddPlugin(std::move(plugin));
 }
 
+void *FirebaseCorePlugin::GetFirebaseApp(std::string appName) {
+  return App::GetInstance(appName.c_str());
+}
+
+void *FirebaseCorePlugin::GetFirebaseAuth(std::string appName) {
```

Comment:
It makes me think that auth will probably need a `tenantId` parameter but I don't think this is available yet on C++.

Reply:
what is the purpose of tenantId? I'd like to see how to map that into the c++ lib.

Reply:
It is sued to determine the tenant of the app: https://cloud.google.com/identity-platform/docs/multi-tenancy